### DISCUSSION TITLE
Add 'Runtime in current epoch' to training gui

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#1266] [Ideas]
Hello,

Sometimes it's difficult to troubleshoot if training on a local machine has stalled (especially if training has been running for ~10 hrs). It would be helpful if in the Training Model Gui's top line, in addition to 'Training Epoch X / Runtime: YY:YY,' there was a 'Runtime in current epoch: ZZ:ZZ'.

Thanks!
Stefan
![image](https://user-images.githubusercontent.com/69451171/230932887-dc2a871b-6c76-4871-9fd2-b3ca3657e973.png)

**Comment by gitttt-1234** [comment]
Closing this discussion, added this in #2705
